Application/EdFi.Ods.Features/OpenApiMetadata/Dtos/OpenApiMetadataDocumentContext.cs

```diff
@@ -37,19 +37,19 @@ public SwaggerDocumentContext(IResourceModel resourceModel)
     }
 
     /// <summary>
-    /// Describes the rendering modes of the SwaggerDocumentFactory
+    /// Describes the rendering modes of the OpenApiMetadataDocumentFactory
     /// </summary>
     public enum RenderType
     {
         /// <summary>
         /// Default rendering mode that includes all resources, with strongly-typed extensions (for a tightly-coupled client-side SDK).
         /// </summary>
         AllWithStronglyTypedExtensions, /// <summary>
-        /// Uses generalized extension collections (for a loosely-coupled client-side SDK).
-        /// </summary>
+                                        /// Uses generalized extension collections (for a loosely-coupled client-side SDK).
+                                        /// </summary>
         GeneralizedExtensions, /// <summary>
-        /// Includes extension resources only and any model definitions for "entity extension" classes.
-        /// </summary>
+                               /// Includes extension resources only and any model definitions for "entity extension" classes.
+                               /// </summary>
```

Comment:
fix this formatting please and revert the white space changes

Reply:
Fixed